Formula/llvm@9.rb

```diff
@@ -169,9 +218,15 @@ def caveats
 
     clean_version = version.to_s[/(\d+\.?)+/]
 
-    system "#{bin}/clang", "-L#{lib}", "-fopenmp", "-nobuiltininc",
-                           "-I#{lib}/clang/#{clean_version}/include",
-                           "omptest.c", "-o", "omptest"
+    args = [
+      "-L#{lib}",
+      "-fopenmp",
+      "-nobuiltininc",
```

Comment:
Passing this on Big Sur might also help with running `test.cpp` on Big Sur, but let's see how this run goes first.